app/components/render-contents/template.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="chapters">
+  <button class='chapter-button' {{action 'expandChapter'}}> {{table.title}} </button>
```

Comment:
I would recommend renaming `table` to something more descriptive like `chapter`. Also, this property should be declared in the `component.js` file and documented so that it's clear that this has to be passed in to the component.